extensions/amp-ad-network-doubleclick-impl/single-page-ad.md

```diff
@@ -39,7 +39,7 @@ You __cannot__ use an `<amp-ad>` element to display a single page ad within a st
 
 ## Single Page Ad Creatives
 
-Single page ad creatives must contain two meta tags: one to specify the [call-to-action enum](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story-ads.md#cta-ad), and one to specify the outlink URL. E.g.,
+Single page ad creatives must contain two meta tags: one to specify the [call-to-action enum](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story-ads.md#cta-enum), and one to specify the outlink URL. E.g.,
```

Comment:
Shouldn't we also add the CTA type enum (AMP, non-AMP or Sponsored Story) as well? CC @lannka

Reply:
agreed with `<meta name="amp-cta-landing-page-type" content="AMP|STORY|NONAMP">`
default is `NONAMP` if the meta tag is missing